applications/stepswitch/src/stepswitch_resources.erl

```diff
@@ -152,7 +153,9 @@ endpoints(Number, OffnetJObj) ->
 -spec maybe_get_endpoints(ne_binary(), kapi_offnet_resource:req()) -> kz_json:objects().
 maybe_get_endpoints(Number, OffnetJObj) ->
     case kapi_offnet_resource:hunt_account_id(OffnetJObj) of
-        'undefined' -> get_global_endpoints(Number, OffnetJObj);
+        'undefined' ->
+            lager:debug("attempting to find global resources"),
+            get_endpoints(get(), Number, OffnetJObj);
```

Comment:
Instead of just `get()`, wouldn't it be better to introduce a special key (e.g. `'global_endpoints'`) that contains a set of keys to endpoints? `get()` at L158 would become `sets:to_list(get('global_endpoints'))` (would need initialisation `put('global_endpoints', sets:new())`).

The intent is to not arbitrarily go through the whole process dictionary.